members/SNJG8473b29e.yaml

```diff
@@ -0,0 +1,144 @@
+bioguide: XXXX
+contact_form:
+  method: post
+  action: /cgi-bin/governor/govmail/govmail_1c_new.pl
+  steps:
+    - visit: "http://www.state.nj.us/governor/contact/"
+    - select:
+        - name: topic
+          selector: form#govmail select.textfield
+          value: $TOPIC
+          required: true
+          options:
+            "Addiction Services": "Addiction Services"
+            "Agriculture": "Agriculture"
+            "Arts and History": "Arts and History"
+            "Banking": "Banking"
+            "Birth/Death Certificates": "Birth/Death Certificates"
+            "Business": "Business"
+            "Children and Families": "Children and Families"
+            "Child Support": "Child Support"
+            "Civil Service Commission": "Civil Service Commission"
+            "Consumer Affairs": "Consumer Affairs"
+            "Corrections": "Corrections"
+            "Developmental Disabilities": "Developmental Disabilities"
+            "Division on Elections": "Division on Elections"
+            "Education": "Education"
+            "Environment": "Environment"
+            "Family Care/Medicaid": "Family Care/Medicaid"
+            "Health": "Health"
+            "Housing": "Housing"
+            "Human Services": "Human Services"
+            "Insurance": "Insurance"
+            "Invitations/Scheduling": "Invitations/Scheduling"
+            "Labor": "Labor"
+            "Law and Public Safety": "Law and Public Safety"
+            "Licensing Boards": "Licensing Boards"
+            "Lottery": "Lottery"
+            "Mental Health Services": "Mental Health Services"
+            "Military and Veterans Affairs": "Military and Veterans Affairs"
+            "Pensions and Retirements": "Pensions and Retirements"
+            "Public Utilities": "Public Utilities"
+            "Requesting Proclamations / Special Letter": "Requesting Proclamations / Special Letter"
+            "Superstorm Sandy/CDBG": "Superstorm Sandy/CDBG"
+            "Taxation": "Taxation"
+            "Temporary Disability": "Temporary Disability"
+            "Transportation / Motor Vehicles": "Transportation / Motor Vehicles"
+            "Unclaimed Property": "Unclaimed Property"
+            "Unemployment": "Unemployment"
+    - click_on:
+        - value: Continue...
```

Comment:
I try to have `click_on` steps only have a `selector` and no `value` - makes the yaml a bit easier to read IMO

Reply:
Okok. So essentially like

- click_on:
  selector: form#govmail ... 

right? 

This click_on + value pair was carried over from Christie's yaml

Reply:
I edited to 
`click_on:
  selector: form#govmail button[name='Submit'] `

but the test failed. Not sure if that yaml code is correct. 

I put `value` back in there and test succeeded.

Reply:
need the dash and correct indentation still

Reply:
yeap, my bad.  Added the dash and the correct indentation and the test passed. Will push the edited yaml now